theano/tensor/elemwise.py

```diff
@@ -185,6 +206,13 @@ def __init__(self, input_broadcastable, new_order, inplace=True):
         if self.inplace:
             self.view_map = {0: [0]}
```

Comment:
I just thought about this, but don't we enable views all the time.  Usually inplace is for destroy_map.

@lamblin ?